docs/detections/prebuilt-rules/downloadable-packages/0-16-2/prebuilt-rule-0-16-2-potential-credential-access-via-duplicatehandle-in-lsass.asciidoc

```diff
@@ -0,0 +1,83 @@
+[[prebuilt-rule-0-16-2-potential-credential-access-via-duplicatehandle-in-lsass]]
+=== Potential Credential Access via DuplicateHandle in LSASS
+
+Identifies suspicious access to an LSASS handle via DuplicateHandle from an unknown call trace module. This may indicate an attempt to bypass the NtOpenProcess API to evade detection and dump LSASS memory for credential access.
```

Comment:
```suggestion
Identifies suspicious access to a Local Security Authority Server Service (LSASS) handle via DuplicateHandle from an unknown call trace module. This may indicate an attempt to bypass the NtOpenProcess API to evade detection and dump LSASS memory for credential access.
```